frontend/src/components/regions/UserActions/ActivityDashboardBarGraph.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+} from "recharts";
+
+import { eventTypeColors } from "../../../colors";
+
+const RED = eventTypeColors.CARD_MOVED_TO_REVIEW_FEEDBACK;
+const GREEN = eventTypeColors.CARD_STARTED;
+const YELLOW = eventTypeColors.CARD_MOVED_TO_COMPLETE;
+
+export default function ActivityDashboardBarGraph({ activityLogDayCounts }) {
+  return (
+    <BarChart
+      width={1415}
+      height={300}
+      data={activityLogDayCounts}
+      margin={{
+        top: 5,
+        right: 30,
+        left: 20,
+        bottom: 5,
+      }}
+    >
+      <CartesianGrid strokeDasharray="3 3" />
+      <XAxis dataKey="date" />
+
+      <Tooltip />
+      <Legend />
+      <Bar dataKey="eventType" fill={RED} name="Project Cards Completed" />
+      <Bar dataKey="eventType" fill={GREEN} name="Topic Cards Completed" />
+      <Bar dataKey="eventType" fill={YELLOW} name="Reviews Completed" />
```

Comment:
I can see that your two branches are already out of sync. Your other branch has:

```
 <Bar
        dataKey="projectCardsCompleted"
        fill={eventTypeColors.CARD_MOVED_TO_COMPLETE}
        date="Project Cards Completed"
      />
      <Bar
        dataKey="topicCardsCompleted"
        fill={eventTypeColors.CARD_MOVED_TO_REVIEW_FEEDBACK}
        date="Topic Cards Completed"
      />
      <Bar
        dataKey="reviewsCompleted"
        fill={eventTypeColors.CARD_STARTED}
        date="Reviews Completed"
      />
```

This is bad:
```
      <Bar dataKey="eventType" fill={RED} name="Project Cards Completed" />
```

- The dataKey is not a useful value. You are graphing event type ids instead of counts
- the fill colour is not explicit and we need to go on a treasure hunt to see if it is the right colour.